builders/manage/sudo/upgrade.md

```diff
@@ -25,13 +26,26 @@ If you're unsure what your Tanssi appchain's Sudo account is, you can find it in
 !!! warning
     You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
 
+## Obtaining the Wasm Runtime {: #obtaining-wasm-runtime }
```

Comment:
Do we say "WASM" or "Wasm"?

Also, I think we need to tell developers that they need to check their current runtime (there are many RPCs to do so, or just check Polkadot.js Apps) and then share a table with all the runtimes with all the URLs because they need to upgrade in order

Reply:
Should be Wasm, per the official site - I think we may need to check and update other pages though

<img width="606" alt="Screenshot 2024-04-28 at 11 27 08 PM" src="https://github.com/moondance-labs/tanssi-docs/assets/26533957/e374ac42-65f0-4fd7-a225-7f41959f8de9">

Reply:
(I also added a new warning about changing the spec name when building a custom Wasm)